examples/quickstart-lerobot/README.md

```diff
@@ -0,0 +1,98 @@
+---
+tags: [robotics, ai, zk0]
+dataset: [pusht]
+framework: [lerobot]
+---
+
+# Federated Learning with HuggingFace LeRobot and Flower (Quickstart Example)
+
+This is an introductory example to using [🤗LeRobot](https://huggingface.co/lerobot) with [🌼Flower](https://flower.ai/). It demonstrates that it is feasible to collaboratively train a robotics AI model in remote environments with their local data and then aggregated it in a shared model.
+
+In this example, we will federate the training of a [Diffusion](https://arxiv.org/abs/2303.04137) policy on the [PushT](https://huggingface.co/datasets/lerobot/pusht/tree/v1.3) dataset. The data will be downloaded and partitioned using [Flower Datasets](https://flower.ai/docs/datasets/). This example runs best when a GPU is available.
+
+## Set up the project
+
+### Clone the project
+
+Start by cloning the example project. We prepared a single-line command that you can copy into your shell which will checkout the example for you:
+
+```shell
+git clone --depth=1 https://github.com/adap/flower.git _tmp \
+		&& mv _tmp/examples/quickstart-lerobot . \
+		&& rm -rf _tmp && cd quickstart-lerobot
+```
+
+This will create a new directory called `quickstart-lerobot` containing the following files:
+
+```shell
+quickstart-lerobot
+├── lerobot_example
```

Comment:
directories `configs` and `policy` are missing

Reply:
I force added them now. Not sure why git excluded earlier. Can you see them?

Reply:
perfect. I see them now 💯